### PR TITLE
refactor(schema): remove "field.Field"

### PR DESCRIPTION
**Pull request by Lee-000**

### CODE REVIEW

**Comment by GravitiRob** [comment]
Task linked: [CU-2840ypd 删除 Field 类型](https://app.clickup.com/t/2840ypd)

File: graviti/schema/factory.py [hunk 1]
```diff
@@ -73,13 +86,10 @@ def __call__(self, **kwargs: Any) -> PTYPE.PType:
 
         """
         ptype = self._ptype_getter(**kwargs)
-        if ptype in {PTYPE.PortexType, PTYPE.Field}:
+        if ptype in {PTYPE.PortexType}:
```
Comment (linjiX) [inline graviti/schema/factory.py:89]: why still using `in`?

File: graviti/schema/factory.py [hunk 2]
```diff
@@ -111,7 +121,7 @@ def __call__(self, **kwargs: Any) -> PTYPE.PType:
 
         """
         if self._ptype_getter(**kwargs) == PTYPE.Fields:
-            return PTYPE.Field
+            return PTYPE.Fields
```
Comment (linjiX) [inline graviti/schema/factory.py:124]: The item of `Fields` should not be `Fields`

File: graviti/schema/factory.py [hunk 3]
```diff
@@ -338,6 +348,9 @@ def __init__(self, decl: List[Any], ptype: PTYPE.PType = PTYPE.Any) -> None:
         dependences = set()
         keys = {}
         ptype = DynamicListParameter(ptype) if isinstance(ptype, Dynamic) else ptype
+        if ptype == PTYPE.Fields:
+            decl = [decl]
```
Comment (linjiX) [inline graviti/schema/factory.py:352]: why we need to do that?

**Review by linjiX — CHANGES_REQUESTED** [review]

File: graviti/schema/factory.py [hunk 4]
```diff
@@ -398,53 +411,6 @@ def __call__(self, **kwargs: Any) -> Dict[str, Any]:
         return {key: factory(**kwargs) for key, factory in self._factories.items()}
 
 
-class FieldFactory(Factory):
```
Comment (linjiX) [inline graviti/schema/factory.py:401]: Why not keep `FieldFactory`?

**Review by linjiX — CHANGES_REQUESTED** [review]

File: graviti/schema/ptype.py [hunk 5]
```diff
@@ -132,44 +131,54 @@ class Field(ParameterType):
...
-            A :class:`Field` instance created by the input argument.
+            A tuple of str and PortexType created by the input argument.
 
         """
-        return ClassField(*arg)
+        try:
+            name, portex_type = arg
+            if isinstance(name, str) and isinstance(portex_type, ClassPortexType):
+                return name, portex_type
+        except ValueError:
```
Comment (linjiX) [inline graviti/schema/ptype.py:151]: why we pass the ValueError?

Comment (Lee-000) [reply graviti/schema/ptype.py:151]: TypeError will be raised instead.

Comment (linjiX) [reply graviti/schema/ptype.py:151]: Is that necessary?

**Review by linjiX — APPROVED** [review]

**Pull request merged**